### PR TITLE
Add em value in select and conversion

### PR DESCRIPTION
**Pull request by anselmh**
Targets #22 and should fix it. Please test it if it works (couldn't test as CORS took effect on my localhost)

### CODE REVIEW

File: app/scripts/sizer-soze.js [hunk 1]
```diff
@@ -140,12 +140,31 @@
     var urlInput = doc.querySelector( "input[name=url]" );
     var breakpoints = doc.querySelectorAll( ".breakpoint" );
     var bps = [];
+    var i = 0;
+    var l = breakpoints.length;
```
Comment (yoavweiss) [inline app/scripts/sizer-soze.js]: Can you give `l` a more meaningful name? Skimming the code, it's easy to confuse `l` `1` and `I`...

Comment (anselmh) [inline app/scripts/sizer-soze.js]: Wasn't introduced by me but I renamed it now.

**Comment by yoavweiss** [comment]
Awesome, thanks! Can you rebase this PR? Once that done, we can merge it.

**Comment by anselmh** [comment]
Is rebased now…

**Comment by yoavweiss** [comment]
Awesome! Thanks! :)

**Pull request merged**